english/the-nights/the-nights.md

```diff
@@ -1,5 +1,5 @@
 One day you'll leave this world behind
-So live a life you will remember.
+So [live a life](../life_well_lived/purpose.md) you will remember.
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.